specification/columns/skupriceid.md

```diff
@@ -21,7 +21,7 @@ A unique identifier that defines the unit price used to calculate the charge.
 | Constraint       | Value          |
 | :--------------- | :------------- |
 | Column type      | Dimension      |
-| Feature level    | Conditional    |
+| Feature level    | Mandatory      |
```

Comment:
Looks like this got screwed up in a previous commit for ChargeCategory fixes. It should say Conditional and also have the conditional sentence in the normative paragraph.

Reply:
@udam-f2, can you confirm whether the condition listed below is correct?

"The SkuPriceId column MUST be present in the billing data when the provider provides a SKU list ...."

Reply:
Looks good